### PR TITLE
doc/user: further clarify the meaning of AS OF/SNAPSHOT in tail

### PR DESCRIPTION
**Pull request by benesch**
As mentioned in #4790. Didn't get around to these improvements before
landing that PR.

---
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/4821)

### CODE REVIEW

File: doc/user/content/sql/tail.md [hunk 1]
```diff
@@ -156,11 +158,16 @@ details on Materialize's compaction policy.
 
 ### `SNAPSHOT`
 
-By default, each `TAIL` begins by emitting a snapshot of the tailed relation,
-which contains the contents of the relation at its [`AS OF`](#as-of) timestamp.
-Any further updates to these results are produced at the time when they occur.
+By default, a `TAIL` begins by emitting a snapshot of the tailed relation, which
+consists of a series of updates describing the contents of the relation at its
+[`AS OF`](#as-of) timestamp. After the snapshot, `TAIL` emits further updates as
```
Comment (sploiselle) [inline doc/user/content/sql/tail.md:163]: Minor consideration: "Any further updates to these results are produced at the time when they occur." from the old state more clearly matches my expectation of the expected behavior, e.g. if `AS OF` is 21, but there are updates up to 30, I would expect 22, 23, etc. to be produced. 

"After the snapshot, `TAIL` emits further updates as they occur." in this edit sounds like in the above e.g. 22 or 23 would be elided from output.

In either case, could I get a little clarification around the expected behavior?

**Review by sploiselle — APPROVED** [review]

File: doc/user/content/sql/tail.md (hunk 1 above)
Comment (benesch) [reply doc/user/content/sql/tail.md:163]: Hah, that was in passive voice so I changed it! Suggestions for an active voice phrasing that's equivalent to the original?

Is it as simple as "after the snapshot, `TAIL` emits updates at the time when they occur."?

Comment (sploiselle) [reply doc/user/content/sql/tail.md:163]: Yeah--that's very simple and clear!

Comment (benesch) [reply doc/user/content/sql/tail.md:163]: Perfect. Done.

**Pull request merged**